src/pages/outlier-detection/OutlierDetection.js

```diff
@@ -33,6 +33,9 @@ const threeMonthsAgo = () => {
     return date
 }
 
+const getMarkedForFollowUpSuccesMessage = marked =>
+    marked ? i18n.t('Unmarked for follow-up') : i18n.t('Marked for follow-up')
```

Comment:
I think this is backwards, actually - the `marked` property has already been inverted when this function is called!

Reply:
I think you've got a point there. I actually had it reversed because that makes much more sense, but that turned out to produce the wrong messages so I simply reversed the messages in eb0ac91. Instead of doing this I should have probably looked at the root cause and fixed that.

Will have a look now.

Reply:
Resolved in 4e6175c

Reply:
Ah it looks like it was actually correct, just confusing - thanks for cleaning it up!